SDWebImage/SDWebImageManager.m

```diff
@@ -108,10 +108,6 @@ - (nullable NSString *)cacheKeyForURL:(nullable NSURL *)url cacheKeyFilter:(id<S
     }
 }
 
-- (nullable UIImage *)scaledImageForKey:(nullable NSString *)key image:(nullable UIImage *)image {
```

Comment:
This private method now is no use at all. Since we scale the image during image decoding and it's never get called.